examples/invoicer/src/providers/color-mode/index.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  PropsWithChildren,
+  createContext,
+  useContext,
+  useEffect,
+  useState,
+} from "react";
+import { ConfigProvider, ThemeConfig, theme } from "antd";
+import { ThemeProvider } from "antd-style";
+import { RefineThemes } from "@refinedev/antd";
+
+type ColorModeContextType = {
+  mode: string;
+  setMode: (mode: string) => void;
+};
+
+export const ColorModeContext = createContext<ColorModeContextType>(
+  {} as ColorModeContextType,
+);
+
+export const ColorModeContextProvider: React.FC<PropsWithChildren> = ({
+  children,
+}) => {
+  const colorModeFromLocalStorage = localStorage.getItem("colorMode");
+  const isSystemPreferenceDark = window?.matchMedia(
```

Comment:
Let's remove if it's not needed.

```suggestion
  const isSystemPreferenceDark = window.matchMedia(
```